business_requirement_deliverable_cost/models/business.py

```diff
@@ -46,10 +46,10 @@ def _compute_sale_price_total(self):
     @api.multi
     def _get_partner(self):
         self.ensure_one()
-        br_id = False
-        if self.business_requirement_deliverable_id:
+        br_id, br_deliverable = 2 * [False]
```

Comment:
looks nice but is quite useless and not efficient... 

* 1 list creation
* another list creation for the result of the multiplication, + garbage collection of the 1st list
* sequence unpacking (with check of the number of elements) for the variable assignment
* garbage collection of the 2nd list

Best way to write this is:

```
br_id = br_deliverable = False
```